.github/workflows/wheels-test.yml

```diff
@@ -152,11 +168,48 @@ jobs:
         date: ${{ inputs.date }}
         sha: ${{ inputs.sha }}
 
-    - name: Set CTK-related vars from input CTK versions
-      uses: rapidsai/shared-action-workflows/wheel-ctk-name-gen@branch-23.06
-      with:
-        ctk: ${{ matrix.ctk }}
-        package-name: ${{ inputs.package-name }}
-
     - name: Run citestwheel
-      run: /citestwheel.sh
+      run: |
+        set -eoxu pipefail
+
+        ctk_tag="${{ matrix.ctk }}"
+        ctk_major=$(echo $ctk_tag | cut -d'.' -f1)
+        cuda_suffix="cu${ctk_major}"
+        wheelname="${{ inputs.package-name }}_cu${ctk_major}"
+
+        export RAPIDS_PY_WHEEL_CUDA_SUFFIX=${cuda_suffix}
+        export RAPIDS_PY_WHEEL_NAME=${wheelname}
```

Comment:
Rather than exporting I think we can probably scope this narrowly to just the `rapids-download-wheels-from-s3` command.